dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataImport.java

```diff
@@ -778,8 +778,8 @@ private void addRelationship(Context c, Item item, String typeName, String value
         }
 
         // Create the relationship
-        int leftPlace = relationshipService.findLeftPlaceByLeftItem(c, leftItem) + 1;
-        int rightPlace = relationshipService.findRightPlaceByRightItem(c, rightItem) + 1;
+        int leftPlace = relationshipService.findNextLeftPlaceByLeftItem(c, leftItem);
```

Comment:
The first time I executed an CSV import, I got an error. On the second try, without any changes, the record was registered. But I noticed a duplication of isAuthorOfPublication on the Metadatafield with the relationship.

`metadata_field_id: 134 (relation.isAuthorOfPublication)`
![2019-10-29 09_52_48-Valentina Studio](https://user-images.githubusercontent.com/877488/67757563-f871e280-fa33-11e9-981d-dcb737c5388a.png)

Should a specific relation.isAuthorOfPublication should be present in this table?

Reply:
Hi @paulo-graca,

I was able to reproduce the problem, but also on master. It seems you've found a bug in one of the CSV branches already merged in master.
Can you move this to a separate ticket so it's not lost?

I was able to reproduce this with the CSV https://www.dropbox.com/s/vvugb1tuknjhp9a/test-records-2.csv?dl=1 which created relation.isAuthorOfPublication with value 0ffbee3f-e7ea-42bc-92fe-2fbef1a52c0f twice using master